### PR TITLE
Update how-to-contribute.md

### PR DESCRIPTION
**Pull request by Ruchitayadav**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by Ruchitayadav** [comment]
Added a new way of contributing to open source

File: _articles/how-to-contribute.md [hunk 1]
```diff
@@ -134,6 +134,12 @@ Even if you like to write code, other types of contributions are a great way to
 * Write tutorials for how a project can be used
 * Offer to mentor another contributor, [like @ereichert did for @bronzdoc on Rust](https://github.com/rust-lang/book/issues/123#issuecomment-238049666)
 
+### Do you like to run the code?
+* Find new bugs
+* New ideas can be fabricated 
+* Learn new ways to write the code as well as comments
+
+
```
Comment (import-brain) [inline _articles/how-to-contribute.md:142]:
```suggestion
### Do you like to run the code?

* Find new bugs
* New ideas can be fabricated 
* Learn new ways to write the code as well as comments

```
This should fix the linting issues, @ me if it doesn't :)

**Review by 1DOKE3 — COMMENTED** [review]
231

**Comment by MikeMcQuaid** [comment]
Passing on this, sorry.

**Pull request closed without merging**